toolkit/tools/imagegen/configuration/disk_test.go

```diff
@@ -72,6 +72,21 @@ func TestShouldSucceedParsingValidDisk_Disk(t *testing.T) {
 	assert.Equal(t, validDisk, checkedDisk)
 }
 
+func TestShouldFailParsingDiskWithBadPartition_Disk(t *testing.T) {
```

Comment:
nit - this is testing for a bad PartitionFlag specifically
```suggestion
func TestShouldFailParsingDiskWithBadPartitionFlag_Disk(t *testing.T) {
```